config/locales/en.yml

```diff
@@ -1016,8 +1016,8 @@ idv.errors.incorrect_password: The password you entered is not correct.
 idv.errors.pattern_mismatch.ssn: Enter a nine-digit Social Security number
 idv.errors.pattern_mismatch.zipcode: Enter a 5 or 9 digit ZIP Code
 idv.errors.pattern_mismatch.zipcode_five: Enter a 5 digit ZIP Code
-idv.failure.attempts_html.one: For security reasons, you have <strong>one attempt</strong> remaining to add your ID online.
-idv.failure.attempts_html.other: For security reasons, you have <strong>%{count} attempts</strong> remaining to add your ID online.
+idv.failure.attempts_html.one: You can try <strong>1 more time.</strong> Then, you must wait 6 hours before trying again.
+idv.failure.attempts_html.other: You can try <strong>%{count} more times.</strong> Then, you must wait 6 hours before trying again.
```

Comment:
Since this is based on a configuration value that could change (`doc_auth_attempt_window_in_minutes`), I think we should probably have interpolate the timespan, possibly using `distance_of_time_in_words` ([docs](https://api.rubyonrails.org/v7.1.3.4/classes/ActionView/Helpers/DateHelper.html#method-i-distance_of_time_in_words), extended with [`dotiw` gem](https://github.com/radar/distance_of_time_in_words)).

Reply:
Ticketing this

Reply:
Ticketed: https://cm-jira.usa.gov/browse/LG-14685